startup/20-area-detectors.py

```diff
@@ -63,8 +139,33 @@ def setExposureTime(self, exposure_time, verbosity=3):
         caput('XF:11BMB-ES{Det:SAXS}:cam1:AcquireTime', exposure_time)
         caput('XF:11BMB-ES{Det:SAXS}:cam1:AcquirePeriod', exposure_time+0.1)
 
+class PilatusV33(SingleTriggerV33, PilatusDetector):
+    cam = Cpt(PilatusDetectorCamV33, 'cam1:')
+    image = Cpt(ImagePlugin, 'image1:')
+    stats1 = Cpt(StatsPluginV33, 'Stats1:')
+    stats2 = Cpt(StatsPluginV33, 'Stats2:')
+    stats3 = Cpt(StatsPluginV33, 'Stats3:')
+    stats4 = Cpt(StatsPluginV33, 'Stats4:')
+    stats5 = Cpt(StatsPluginV33, 'Stats5:')
+    roi1 = Cpt(ROIPlugin, 'ROI1:')
+    roi2 = Cpt(ROIPlugin, 'ROI2:')
+    roi3 = Cpt(ROIPlugin, 'ROI3:')
+    roi4 = Cpt(ROIPlugin, 'ROI4:')
+    proc1 = Cpt(ProcessPlugin, 'Proc1:')
+
+
+    tiff = Cpt(TIFFPluginWithFileStore,
+               suffix='TIFF1:',
+               write_path_template='/GPFS/xf11bm/Pilatus300/%Y/%m/%d/',
+               root='/GPFS/xf11bm')
+
+    def setExposureTime(self, exposure_time, verbosity=3):
+        caput('XF:11BMB-ES{Det:SAXS}:cam1:AcquireTime', exposure_time)
```

Comment:
As long as you are touching these, this should be `self.cam.acquire_time.put()` and `self.cam.acquire_period.put()`.

Also inclined to put `warnings.warn("turn where ever you are using this into a plan", stacklevel=2)` in as well, but that is for a later day...